lib/ridgepole/external_sql_executer.rb

```diff
@@ -48,5 +48,15 @@ def execute(sql)
         raise "`#{@script}` execution failed" unless wait_thr.value.success?
       end
     end
+
+    private
+
+    def connection_configuration_hash
+      if ActiveRecord.gem_version >= Gem::Version.new('6.1.0')
+        ActiveRecord::Base.connection_db_config.configuration_hash
+      else
+        ActiveRecord::Base.connection_config
+      end
```

Comment:
I think it's better to change the conditions to truncate the old version eventually.

```suggestion
      if ActiveRecord.gem_version < Gem::Version.new('6.1.0')
        # NOTE: Remove code when stopping support for versions below 6.1
        ActiveRecord::Base.connection_config
      else
        ActiveRecord::Base.connection_db_config.configuration_hash
      end
```